stable/opsportal/values.yaml

```diff
@@ -49,3 +49,55 @@ opsportal:
           serviceName: traefik-kubeaddons-dashboard
           servicePort: 80
         path: /ops/portal/traefik
+
+
+###############################################################################
+#  Konvoy UI
+###############################################################################
+
+image:
+  repository: mesosphere/kommander
+  tag: 1.60.0
+  pullPolicy: IfNotPresent
+replicas: 1
+logoutRedirectPath: /ops/landing
+clusterPollingInterval: 3000
+clientPollingInterval: 3000
+
+# Mode must be either production|konvoy
+mode: konvoy
+
+displayName: Konvoy Cluster
+
+# Konvoy ui should NOT create the CRD if Kommander already did it
+createObservableClusterCRD: true
```

Comment:
hmm... can we take some of the configs `konvoy-ui` in `kubeaddons-configs` to be the defaults here?

Reply:
I think I took everything it is there 🤔  ... please let me know if I missed any value

Reply:
`Konvoy ui should NOT create the CRD if Kommander already did it` this comment was misleading. it seems like you have all the values.

Reply:
let's remove that coment.

Reply:
I took the defaults from kubeaddons-configs

Reply:
is only the comment something we want to remove ? or there is anything else... please do a more specific review and point the values and properties. We want to have the defaults from kubeaddons-configs here, afaik.

Reply:
think that's all we need to remove.